### PR TITLE
[IMP] payment{,_stripe},{,website_}sale: clean up stripe connect code

### PR DESCRIPTION
**Pull request by tlibioulle**
This commit removes deprecated actions and methods kept after the merge
of PR #79621 in stable.

Follow-up
task-2685160

References
---------------

- Enterprise: https://github.com/odoo/enterprise/pull/25369
- https://github.com/odoo/upgrade/pull/3256

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by AntoineVDV** [comment]
We should also remove the `auto_install` of `payment_transfer`.

**Review by AntoineVDV — APPROVED** [review]
I pushed the last little cleanups that I could think about. I think you took care of everything else so we should be ready to merge after you handle `website_delivery_ups` and squash the commits :)

File: addons/website_sale/views/onboarding_views.xml [hunk 1]
```diff
@@ -1,18 +1,14 @@
 <odoo>
-    <!-- PAYMENT ACQUIRER -->
-    <template id="website_sale_onboarding_payment_acquirer_step" primary="True"
-              inherit_id="payment.onboarding_payment_acquirer_step">
-        <xpath expr="//t[@t-set='method']" position="replace">
+    <!-- PAYMENT ACQUIRER - TLE : Move it to website_sale_dashboard ? -->
```
[the remark this answers is not in the document]
Comment (tlibioulle) [reply addons/website_sale/views/onboarding_views.xml]: Applied :)

**Comment by AntoineVDV** [comment]
@robodoo r+

**Comment by AntoineVDV** [comment]
@robodoo delegate+

**Comment by tlibioulle** [comment]
@robodoo r+

**Comment by tlibioulle** [comment]
@robodoo r+
:monocle_face:

**Comment by AntoineVDV** [comment]
@tlibioulle It's on the upgrade PR that you need a new r+ cc @KangOl

**Comment by tlibioulle** [comment]
> @tlibioulle It's on the upgrade PR that you need a new r+ cc @KangOl

@AntoineVDV In fact there was an issue with the mergebot which didn't know this PR was reviewed :) (FYI)

**Comment by tlibioulle** [comment]
@robodoo r+
:monocle_face: Once again

**Pull request closed without merging**